tests/rt.conf

```diff
@@ -3,23 +3,25 @@
 ###################################################################################################################################################################################
 
 COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1                                                           | - wcoss_cray                            | fv3 |
-# Waves off
 RUN     | cpld_control_p7                                                                                                         | - wcoss_cray                            | fv3 |
-RUN     | cpld_restart_p7                                                                                                         | - wcoss_cray                            |     | cpld_control_p7
 RUN     | cpld_2threads_p7                                                                                                        | - wcoss_cray                            |     |
 RUN     | cpld_decomp_p7                                                                                                          | - wcoss_cray jet.intel                  |     |
-# Waves on
-RUN     | cpld_control_wave_p7                                                                                                    | - wcoss_cray                            | fv3 |
+RUN     | cpld_mpi_p7                                                                                                             | - wcoss_cray                            |     |
+
+RUN     | cpld_bmark_p7                                                                                                           | - wcoss_cray                            | fv3 |
+RUN     | cpld_bmark_mpi_p7                                                                                                       | - wcoss_cray                            |     |
```

Comment:
Why do we need an mpi test for both the default and bmark resolutions?

Reply:
The "cpld_bmark_mpi_p7" test was added to show that this bug was fixed as it is not apparent in the "cpld_control_wave_p7" test.  Without this test we could not debug the stated issue or catch the problem.  That being said, it's not typical to have this test.  It can be removed if deemed necessary for the final PR.

Reply:
I read through the issue [317](https://github.com/NOAA-EMC/WW3/issues/317) and it isn't clear to me whether the explanation for why this didn't impact the low-resolution test but did impact the bmark configuration was found. I'm wondering if testing at low-resolution will be adequate?

Reply:
I believe we just got lucky that we do not see it on the C96 test case, essentially during the forecast time, we do not have a point that has enough ice growth during the forecast to become a "deactivated" sea-point and therefore trigger the issue. It was either a little luck, forecast length, IC date or function of resolution or all of the above.

Reply:
So is the MPI test for both resolutions required, in your opinion?

Reply:
I assumed I'd be asked to delete this test, but wanted to include it in the initial PR at the minimum to show we accomplished our goal and on the off-chance others thought it should stay.  Testings non C96 low resolution for the operational requirements is not a part of the regression tests and I understand we have to draw the line of when/where to add the tests.  I also think that if the operational requirements test was expanded to support running the higher resolution cases, that could be sufficient.   I think the biggest argument for inclusion of this would be that different physics are used in the ocean model, but that's never been enough to add one in the past, so I don't see the wave model's situation would pass that threshold either.

Reply:
OK, thanks. I have the same feeling about the longer c192 test; we initially created a long test for a specific issue that Moorthi had reported, but we've continued to carry it. It is hard to drop a test once it has been added. 

I think it might beIt useful in this case to commit the test just to mark a place where we knew something was working.